Documentation/network/bgp-control-plane.rst

```diff
@@ -286,6 +286,60 @@ io.kubernetes.service.namespace ``.meta.namespace``
 io.kubernetes.service.name      ``.meta.name``
 =============================== ===================
 
+CLI
+---
+
+There are two CLIs available to view cilium BGP peering state. One is present
+inside cilium agent. And second is cluster wide cilium-cli (Experimental).
+
+Cilium Agent CLI
+~~~~~~~~~~~~~~~~
+
+Following command will show peering status.
+
+.. code-block:: shell-session
+
+   cilium# cilium bgp peers -h
+   List state of all peers defined in CiliumBGPPeeringPolicy
+
+   Usage:
+     cilium bgp peers [flags]
+
+   Flags:
+     -h, --help            help for peers
+     -o, --output string   json| yaml| jsonpath='{}'
+
+   Global Flags:
+         --config string   Config file (default is $HOME/.cilium.yaml)
+     -D, --debug           Enable debug messages
+     -H, --host string     URI to server-side API
+
+
+Cilium-CLI
+~~~~~~~~~~
+
+Cilium-cli will display BGP peering status of all nodes. 
+
+.. code-block:: shell-session
+
+   # cilium-cli bgp peers -h
+   Gets BGP peering status from all nodes in the cluster
```

Comment:
nits
```suggestion
   This command gets the BGP peering status from all nodes in the cluster.
```

Reply:
Same as above, this is part of CLI output.

Reply:
As mentioned above, I'll accept "sorry, it's upstream" this time. I am considering whether to allow this in the future, on the basis that without blocking feedback, no one faces pressure to fix bad upstream docs.